components/landing/navigation.tsx

```diff
@@ -7,18 +7,20 @@ import { TailspinLogo } from "../ui/spinning-logo";
 export const Navigation = () => {
     return (
         <nav className='fixed top-0 z-[1] flex h-16 w-full items-center'>
-            <Link href='#landing'>
+            <Link href='/#landing'>
```

Comment:
Typo?

Reply:
So, as I carry the navbar to the 404 and 401 page just incase the user wants to use that to navigate back (even though we have a redirect and a button) they can do so.

Without the backslash, the link won't direct to the root page, as there is no path being directed to.